website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -65,16 +65,15 @@ IDs.](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html)
 
 ### Capacity Type
 
-- key: `node.k8s.aws/capacity-type`
+- key: `karpenter.sh/capacity-type`

```

Comment:
We need to be a lil' careful on updating docs before releasing.

Reply:
I'm not following your comment here. Are you saying I shouldn't update the docs? Or is there some other issue with my doc update?

This change is essentially backwards incompatible so it's important IMO that the docs match the functionality.

Reply:
The docs are published immediately when the PR is merged, but we won't release a new version immediately when merged. So the docs will be updated with the new `karpenter.sh/capacity-type` but our latest published container image will not include it.  Anyone following the docs between that time will get errors.  

It might be better to open a second PR w/ the docs changes and label it with the release to coordinate the docs release at the same time as the artifact releases?  We don't have a great way of handling it today :(

Reply:
Ah, that's a PITA. Should we have a doc branch then? I don't know what to do with this second PR exactly - what does approval of it mean, because we can't pull it until we're ready to cut a release, right? And then, if there are multiple doc changes from multiple PRs someone has to go through the fun exercise of resolving conflicts. Am I missing something?

Is there some special syntax with labeling the commit? Do you mean just putting v0.5 in the commit message? Or do you mean adding the label in the PR in the GitHib UI? If so, I also have no idea how to do this - on this very PR on the right side the labels say "None yet" and there's no edit button or anything.

I'll back this change out, but I don't know how we make sure everyone currently contributing to Karpenter knows about this in the interim because it is a backwards incompatible change.

Reply:
Re: labeling - am I supposed to do something like this?

`git tag v0.4.2 <commit_sha>`

Reply:
I meant in the github UI, you should have labeling permissions. Maybe we should just cut a v0.4.3 release when this gets merged since we are in a state of rapid iteration here.

Reply:
Pretty sure I don't have those permissions, how can I get them? I should already be in awslabs.

Reply:
@eptiger I just made you an Admin on this repo, so hopefully that'll work?

Reply:
Oh Brandon gave me the permissions I needed, but I don't see a label for v0.4.3 yet - should I create one?